### PR TITLE
FALCON-2060 Retry does not happen if instance timedout

### PR DESCRIPTION
**Pull request by pallavi-rao**
These got broken when we moved from wf rerun to coord rerun.

### CODE REVIEW

File: oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java [hunk 1]
```diff
@@ -1603,6 +1603,8 @@ public String getWorkflowStatus(String cluster, String jobId) throws FalconExcep
             } else if (jobId.endsWith("-B")) {
                 BundleJob bundle = client.getBundleJobInfo(jobId);
                 return bundle.getStatus().name();
+            } else if (jobId.contains("-C@")) {
```
Comment (peeyushb) [inline oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: Can jobId contains "-C@", I think coordinator action/instance id contains "-C@" . Please confirm.

Comment (pallavi-rao) [inline oozie/src/main/java/org/apache/falcon/workflow/engine/OozieWorkflowEngine.java]: jobId is just a variable name. We are passing bundles, coordinators, workflows and coord actions to this method. Yes. -C@ is applicable only for coord action instance.

**Comment by peeyushb** [comment]
Thanks @pallavi-rao for clarification. 

Patch looks good. +1.

**Comment by sandeepSamudrala** [comment]
:+1:

**Comment by pallavi-rao** [comment]
Thanks @peeyushb, @sandeepSamudrala for the review. Merging as this is a blocker for 0.10.

**Pull request closed without merging**